code/firmware/rosco_m68k_firmware/stage2/load/load.c

```diff
@@ -15,17 +15,39 @@
 
```

Comment:
So, looking at this, I _think_ the elf loading _replaces_ the flat binary loader, is that right?

Reply:
It doesn't fully replace the flat binary loading, it just adds another option. At line 315, it adds a second file to check for and try to load. It will still first try to load `ROSCODE1.BIN` as a flat binary, but will then try to read `ROSCODE1.ELF` an ELF file.

Reply:
I see, that makes more sense to me now, thanks! 👍